package.json

```diff
@@ -411,5 +446,64 @@
     "webpack-node-externals": "^3.0.0",
     "xterm": "^4.19.0",
     "xterm-addon-fit": "^0.5.0"
+  },
+  "peerDependencies": {
+    "@async-fn/jest": "1.6.4",
+    "@material-ui/core": "^4.12.3",
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.60",
+    "@sentry/types": "^6.19.7",
+    "@types/byline": "^4.2.33",
+    "@types/chart.js": "^2.9.36",
+    "@types/circular-dependency-plugin": "5.0.5",
+    "@types/cli-progress": "^3.11.0",
+    "@types/color": "^3.0.3",
+    "@types/command-line-args": "^5.2.0",
+    "@types/crypto-js": "^3.1.47",
+    "@types/dompurify": "^2.4.0",
+    "@types/electron-devtools-installer": "^2.2.1",
+    "@types/fs-extra": "^9.0.13",
+    "@types/glob-to-regexp": "^0.4.1",
+    "@types/gunzip-maybe": "^1.4.0",
+    "@types/hapi__call": "^9.0.0",
+    "@types/hapi__subtext": "^7.0.0",
+    "@types/html-webpack-plugin": "^3.2.6",
+    "@types/http-proxy": "^1.17.9",
+    "@types/jest": "^28.1.6",
+    "@types/js-yaml": "^4.0.5",
+    "@types/jsdom": "^16.2.14",
+    "@types/lodash": "^4.14.191",
+    "@types/marked": "^4.0.8",
+    "@types/md5-file": "^4.0.2",
+    "@types/memorystream": "^0.3.0",
+    "@types/mini-css-extract-plugin": "^2.4.0",
+    "@types/mock-fs": "^4.13.1",
+    "@types/node": "^16.18.9",
+    "@types/proper-lockfile": "^4.1.2",
+    "@types/randomcolor": "^0.5.7",
+    "@types/react": "^17.0.45",
+    "@types/react-beautiful-dnd": "^13.1.3",
+    "@types/react-dom": "^17.0.16",
+    "@types/react-router": "^5.1.19",
+    "@types/react-router-dom": "^5.3.3",
+    "@types/react-table": "^7.7.12",
+    "@types/react-virtualized-auto-sizer": "^1.0.1",
+    "@types/react-window": "^1.8.5",
+    "@types/readable-stream": "^2.3.13",
+    "@types/request": "^2.48.7",
+    "@types/request-promise-native": "^1.0.18",
```

Comment:
Hard to see that all of these are actually `peerDependencies`. `peerDependency` is something that is required to be installed when using already compiled library and e.g. `@async-fn/jest` is only used for development. 

There might be something that I don't see here and I don't think this is that important yet, so let's figure out these later.

Reply:
The thing is that when a library is compiled it won't include dependent declarations (https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#packaging-dependent-declarations). I can try to narrow down which of the types can be removed from peerDependencies. It's also a bit unfortunate that we need to duplicate these (dev vs peer), switching from yarn to npm might make this a bit cleaner.

Reply:
I'd say it's fine for now. Let's clean these later.

Reply:
> switching from yarn to npm might make this a bit cleaner.

How so? For curious am I ;)

Reply:
> > switching from yarn to npm might make this a bit cleaner.
> 
> How so? For curious am I ;)

NPM >= 7 installs peerDependencies automatically, yarn does not allow to install local peerDeps. I think this would mean that we would not need to duplicate packages to both dev & peer.

Reply:
Fixed to include minimal list of peerDependencies (without these client/consumer will start to spit type errors).